<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>S_AES</class>
 <widget class="QMainWindow" name="S_AES">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>520</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>S-AES算法</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SNameLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>66</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>S-AES加密机</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="GroupName">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>xxx组</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>631</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="TextLabel">
         <property name="text">
          <string>明密文：          </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TextlineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="EncryptButton">
         <property name="text">
          <string>加密</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="KeyLabel">
         <property name="text">
          <string>原始密钥：        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="KeylineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="DecrypthButton">
         <property name="text">
          <string>解密</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="ModelLabel">
         <property name="text">
          <string>模式选择：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>普通模式</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>双重加密</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_3">
         <property name="text">
          <string>三重加密</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="IVLabel">
         <property name="text">
          <string>初始向量：        </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IVlineEdit"/>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_4">
         <property name="text">
          <string>CBC模式</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="MessageBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
